README.md

```diff
@@ -1,6 +1,6 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# OOP School Library
 
 > Description the project.
 
```

Comment:
- [ ] Kindly update your README file, update the `Built With` section, and the `Authors` section adds your links, and any other section you can customize.